RMVtransport/rmvtransport.py

```diff
@@ -80,12 +84,11 @@ async def get_departures(
         try:
             self.now = self.current_time()
             self.station = self._station()
-        except (TypeError, AttributeError, ValueError) as err:
+        except RMVtransportDataError:
             _LOGGER.debug(
-                f"Time/Station TypeError or AttributeError {err} "
-                f"{objectify.dump(self.obj)[:100]}"
+                f"XML contains unexpected data " f"{objectify.dump(self.obj)[:100]}"
```

Comment:
```suggestion
                f"XML contains unexpected data {objectify.dump(self.obj)[:100]}"
```

Reply:
Great when back hits and you don't check it's changes. Thanks for taking a look.